src/quo2/components/list_items/address/view.cljs

```diff
@@ -0,0 +1,65 @@
+(ns quo2.components.list-items.address.view
+  (:require [quo2.components.avatars.wallet-user-avatar :as wallet-user-avatar]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.list-items.address.style :as style]
+            [reagent.core :as reagent]
+            [clojure.string :as string]))
+
+(defn- left-container
+  [{:keys [theme address networks blur?]}]
+  [rn/view {:style style/left-container}
+   [wallet-user-avatar/wallet-user-avatar
+    {:size                :medium
+     :f-name              "0"
+     :l-name              "x"
+     :monospace?          true
+     :uppercase?          false
+     :customization-color :neutral}]
+   [rn/view {:style style/account-container}
+    [text/text {:size :paragraph-1}
+     (map (fn [network]
+            ^{:key (str network)}
+            [text/text
+             {:size   :paragraph-1
+              :weight :semi-bold
+              :style  {:color (get colors/networks network)}} (str (subs (name network) 0 3) ":")])
```

Comment:
perhaps we can just pass in 3 character network names instead and we will have a subscription that does this instead for us? 🤔

Reply:
oh btw with network colors I already have the solution in place. it's to use custom-color function to resolve the network color. 👍 anyway I will update this in my colors pr 👍 
`(get colors/networks network` -> `(colors/custom-color  network)`